scripts/deploy/package-webapi.sh

```diff
@@ -72,7 +72,7 @@ if [[ ! -d "$PUBLISH_ZIP_DIRECTORY" ]]; then
 fi
 
 echo "Build configuration: $CONFIGURATION"
-dotnet publish "$SCRIPT_ROOT/../webapi/CopilotChatWebApi.csproj" --configuration $CONFIGURATION --framework $DOTNET --runtime $RUNTIME --self-contained --output "$PUBLISH_OUTPUT_DIRECTORY"
+dotnet publish "$SCRIPT_ROOT/../../webapi/CopilotChatWebApi.csproj" --configuration $CONFIGURATION --framework $DOTNET --runtime $RUNTIME --self-contained --output "$PUBLISH_OUTPUT_DIRECTORY"
 if [ $? -ne 0 ]; then
```

Comment:
general -- is there a plan to move the file naming to chat copilot?

Reply:
I don't think so but it makes sense to do that in this PR since it's already a big structure change to the webapi

Reply:
I'd like to keep these changes separate from the rename if you all are ok with it. 
I will follow-up this PR with another to rename things across the board (in both webapi and webapp): #102

Reply:
makes sense to me!

Reply:
Same!